package.json

```diff
@@ -2,11 +2,12 @@
   "name": "virtual-jade-loader",
   "version": "0.2.2",
   "description": "webpack loader for jade -> virtual-dom/hyperscript",
-  "main": "index.js",
+  "main": "ilib/index.js",
   "engines": {
     "node": ">=4.2"
   },
   "scripts": {
+    "lint": "node_modules/eslint/bin/eslint.js ./lib",
```

Comment:
it's something i've been doing so that the developer doesn't have to have `eslint` installed globally, thoughts?